Tools/autotest/arducopter.py

```diff
@@ -672,248 +687,263 @@ def fly_throttle_failsafe(self, side=60, timeout=360):
         
     # Tests all actions and logic behind the GCS failsafe
     def fly_gcs_failsafe(self, side=60, timeout=360):
-        # Trigger telemety loss with failsafe disabled. Verify no action taken.
```

Comment:
If we do need to wrap this in a try - could we break out a method and wrap that instead of doing this massive reindenting?

Reply:
done